### DISCUSSION TITLE
Qtile on mobile

### DISCUSSION THREAD

**Discussion by zordsdavini** [elParaguayo/qtile-extras#373] [Ideas]
There was try to move Qtile into mobile. I think based on sxmo. As sxmo is super hacky, maybe Qtile could suggest its own way with widgets and popups. 

Libsgd is the gesture deamon what could be configured as Qtile extension. 

Question is, maybe you know old work on this? Maybe there were some decision made?

**Comment by elParaguayo** [comment]
Why are you posting this on qtile-extras, not on qtile? I'm also not clear what you're asking - I'm not aware of any attempt to port qtile to mobile.

**Reply by zordsdavini** [reply]
I thought what this could use a lot of stuff from this repo and changes would not fit into main qtile. Never mind - I repost question to qtile.

**Reply by elParaguayo** [reply]
This repo is more about enhancing the basic functionality of qtile, not rewriting it for other platforms.

**Reply by zordsdavini** [reply]
Support for gestures would be enhancment :) But as extension it can go to base
